tracer/test/snapshots/Security.AspNetMvc5.Classic.enableSecurity=False.blockingEnabled=False.__url=_Health_wp-config.verified.txt

```diff
@@ -0,0 +1,222 @@
+[
+  {
+    TraceId: Id_1,
+    SpanId: Id_2,
+    Name: aspnet-mvc.request,
+    Resource: GET /health/wp-config,
+    Service: sample,
+    Type: web,
+    ParentId: Id_3,
+    Tags: {
+      aspnet.action: wp-config,
+      aspnet.controller: health,
+      aspnet.route: {controller}/{action}/{id},
+      env: integration_tests,
+      http.method: GET,
+      http.request.headers.host: localhost:00000,
+      http.status_code: 200,
+      http.url: /health/wp-config,
+      language: dotnet,
+      span.kind: server
+    }
+  },
+  {
```

Comment:
Should this really include 5 traces with 2 spans each 🤔

Reply:
We send [5 requests](https://github.com/DataDog/dd-trace-dotnet/blob/9c8123469b728fe24cd9d5c43bb90fef7da9e703/tracer/test/Datadog.Trace.Security.IntegrationTests/AspNetBase.cs#L96) everytime so I think it's quite normal 🤔  (the tracing module and the mvc observer?)